enactjs/enact
# ENYO-3994: VideoPlayer no longer renders 2 copies of its children

Fixed by turning `source` into a slot, and updating slot tables to forward nodes with multiple props directly to the component's prop.

## To Test
Simply Inspect and verify that there are only `<source>` tag(s) inside the `<video>` element and that there are no `<source>` tags in the "more" controls area.
